.github/actions/people/people.py

```diff
@@ -570,6 +570,18 @@ def get_top_users(
         skip_users=skip_users,
     )
 
+    extra_experts = [
+        {
+            "login": "ybressler",
+            "count": None,
+            "avatarUrl": "https://avatars.githubusercontent.com/u/40807730?v=4",
+            "url": "https://github.com/ybressler"
+        },
+    ]
+    for expert in extra_experts:
+        if expert["login"] not in {e["login"] for e in experts}:
```

Comment:
Better to create the expers set `{e["login"] for e in experts}` outside of the loop